CONTRIBUTOR-ROLES.md

```diff
@@ -0,0 +1,122 @@
+    * Representing the Cilium project positively in public
+    * Coordinating people, projects, and activities across the community team
+
+* Qualifications:
+    * Organization Member
+    * Sustained contributions to growing the Cilium community through content, promotion, or other ideas
+    * Proven ability to write effective content 
+
+* Privileges:
+    * Access to Cilium social media accounts and mailing lists
+
+The process of becoming a Community Team member is:
+
+1. Make a PR against the community team file
+2. Approval by one committer
+
+### CI Team
+The CI team works to keep the CI infrastructure up and running.
+
+* Responsibilities:
+    * Keeping CI infrastructure healthy
+    * Investigating CI infrastructure problems
+    * Provide visibility into CI spend and status
+
+* Qualifications:
+    * Must be a Cilium Org member
+    * A basic understanding of how the CI system works
+
+* Privileges:
+    * Access rights to configuring CI infrastructure
+
+The process of becoming a CI Team member is:
+
+1. Make a PR against the CI Team file
+2. Approval by one committer
+
+### Code Team
+The Code Team is the committers responsible for all aspects of the Cilium organization codebase. All committers contributing code are automatically a part of this team.
+
+Responsibilities:
+     * Is an active owner of code they have contributed (unless ownership is explicitly transferred) making sure that code is well tested, tests consistently pass, and addressing bugs or issues discovered after code is accepted
+    * Everything in expectations of developers with [commit access](https://docs.cilium.io/en/latest/community/governance/commit_access/#expectations-for-developers-with-commit-access)
+    * Nominate themselves to review specific code owner areas
+    * Backport fixes for latest release
+    * Writing and/or refactoring PRs
+    * Can drive the release process
+    * Help contributors get PRs into a `ready-to-merge` state
+    * Approve CFPs
+
+* Qualifications:
+  * Cilium Committer
+
+### Maintainer
+Maintainers are very established contributors who are responsible for the entire project. They have the same rights as Committers, but have additional administrative and janitorial responsibilities.
+
+A Maintainer must meet the responsibilities and qualifications of a Committer, plus:
+
+* Responsibilities:
+    * Driving the release process
+    * Merging PRs into the main tree
+    * Define release timeline
+    * Other duties as needed by the project
+
+* Qualifications
+    * Demonstrates a broad knowledge of the project across multiple areas
+    * Must be a Cilium committer
+
+* Privileges:
+    * Access and the ability to modify secrets used to publish project images
+    * Define branch protections for stable branches
+
+The process of becoming a Maintainer is:
+    * Talk to the current maintainers about which part of the project you would like to help with.
```

Comment:
Please review and harmonise indent level for first-level and second-level list items, making sure you don't indent too much so that it remains Markdown lists and not pre-formatted blocks.

```suggestion
* Talk to the current maintainers about which part of the project you would like to help with.
```